elm/compiler
# update npm installers for 0.19.0

Local test procedure:

1. make a shallow clone of my fork: `git clone --depth 1 --branch npm-installers-for-0.19 git@github.com:BrianHicks/compiler.git`
2. install the compiler: `npm install compiler/installers/npm`
3. verify you can run the compiler: `npx elm --help`

AND verify that you can do the same with scripts disabled:

1. `git clone --depth 1 --branch npm-installers-for-0.19 git@github.com:BrianHicks/compiler.git`
2. install the compiler: `npm install compiler/installers/npm --ignore-scripts true`
3. verify that `./node_modules/.bin/elm` has a comment stanza at the top about `--ignore-scripts`
3. verify you can run the compiler: `npx elm --help`

@rtfeldman and I have done this locally on Mac and it works fine there. 😄
